addons/mail/static/src/models/attachment.js

```diff
@@ -195,10 +206,13 @@ registerModel({
          */
         isDeletable: attr({
             compute() {
-                if (!this.messaging) {
-                    return false;
```

Comment:
This condition should be kept intact as it is used for the code below which uses `this.messaging`.